pkgs/applications/audio/ncspot/default.nix

```diff
@@ -6,12 +6,13 @@
 , ncurses
 , openssl
 , Cocoa
-, withALSA ? true, alsa-lib
+, withALSA ? false, alsa-lib
 , withClipboard ? true, libxcb, python3
 , withCover ? false, ueberzug
-, withPulseAudio ? false, libpulseaudio
+, withPulseAudio ? true, libpulseaudio
 , withPortAudio ? false, portaudio
 , withMPRIS ? true, withNotify ? true, dbus
+, withCross ? true
```

Comment:
Would you mind changing this to `withCrossterm`?
Besides that, LGTM. Thanks!

Reply:
Sure, thanks for the review :)